src/main/scala/edu/tulane/cs/hetml/nlp/sprl/VisualTriplets/VisualTripletsApp.scala

```diff
@@ -63,10 +57,11 @@ object VisualTripletsApp extends App {
 //      )
 //    }
 
-    visualTriplets.populate(externalTrainTriplets)
-    VisualTripletClassifier.learn(50)
-//    VisualTripletOnClassifier.learn(50)
-//    VisualTripletInClassifier.learn(50)
+    visualTriplets.populate(vggFrequentTriplets)
+//    VisualTripletClassifier.learn(50)
+//    VisualTripletClassifier.save()
+    VisualTripletClassifier.load()
+    VisualTripletClassifier.test(visualTriplets())
 
     //fine tune
```

Comment:
if you need to keep this old case make several options to be able to select the way we can train, otherwise (if you do not need this set-up) remove the commented lines.